content/applications/sales/sales/sales_quotations.rst

```diff
@@ -0,0 +1,138 @@
+from initial interest to payment.
+
+The typical flow follows these steps:
+
+#. *Quotation* : A proposal sent to the customer with product details and pricing.
+#. *Sales order*: Created automatically when the customer accepts the quotation, confirming the sale.
+#. *Delivery* (if applicable): Products are shipped or services are delivered.
+#. *Invoice*: The final bill is issued based on the sales order or delivered products/services.
+#. *Payment*: The customer settles the invoice, completing the sales cycle.
+
+This flow helps businesses track the entire lifecycle of a sale while keeping information consistent
+across apps.
+
+In Odoo, quotations are configured in the **Sales** app. They can also be generated from other apps
+as part of the sales workflow:
+
+- CRM: :doc:`Convert opportunities <../crm/acquire_leads/send_quotes>` into quotes to follow up on
+  potential deals.
+- Helpdesk: :doc:`Generate quotes from tickets <../../services/helpdesk/advanced/after_sales>` when
+  offering paid services or products.
+- Subscriptions: :doc:`Offer recurring services <../subscriptions>` before
+  starting an automatic billing cycle.
+
+.. cards::
+
+   .. card:: Create quotations
+      :target: sales_quotations/create_quotations
+
+      Create, configure, and send quotations to customers.
+
+   .. card:: Quotation templates
+      :target: sales_quotations/quote_template
+
+      Configure and use quotation templates to send tailor-fit quotations at a quicker pace.
+
+   .. card:: Optional products
+      :target: sales_quotations/optional_products
+
+      Offer useful and related products to customers to increase sales.
+
+   .. card:: Online signatures for order confirmations
+      :target: sales_quotations/get_signature_to_validate
+
+      Customers have the ability to confirm orders via online signatures, directly on sales orders.
+
+   .. card:: Online payment order confirmation
+      :target: sales_quotations/get_paid_to_validate
+
+      Customers have the ability to confirm orders via online payment, directly on sales orders.
+
+   .. card:: Quotation Deadlines
+      :target: sales_quotations/deadline
+
+      Set deadlines on quotations to encourage customers to act in a timely manner when
+      closing business deals.
+
+   .. card:: Deliver orders and invoices to different addresses
+      :target: sales_quotations/different_addresses
+
+      Specify separate customer delivery and invoicing addresses on quotations.
+
+   .. card:: Product variants on quotations and sales orders
+      :target: sales_quotations/orders_and_variants
+
+      Add product variants to sales orders to provide additional options for single products.
+
+   .. card:: PDF quote builder
+      :target: sales_quotations/pdf_quote_builder
+
+      Add custom PDF files to quotations to elevate the document's headers and designs.
+
+Sales quotations in business deals
```

Comment:
Would cut this section
It looks/sounds important but doesn't contain any actionable knowledge. It's way better that we "show" vs. "tell,"  generally speaking, and also especially on user docs.